src/seedu/addressbook/storage/Encryptor.java

```diff
@@ -0,0 +1,61 @@
+package seedu.addressbook.storage;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+
+/**
+ * Encrypts and decrypts a string using AES
+ */
+public class Encryptor {
+
+    private static String key = "0000000000000000"; // Placeholder key
+
+    /**
+     * Encrypts string using the AES algorithm
+     *
+     * @param toEncrypt string to be encrypted
+     * @return encrypted string
+     */
+    public static String encrypt(String toEncrypt) {
+        try {
+            SecretKeySpec skeySpec = new SecretKeySpec(key.getBytes("UTF-8"), "AES");
+
+            Cipher cipher = Cipher.getInstance("AES/ECB/PKCS5Padding");
+            cipher.init(Cipher.ENCRYPT_MODE, skeySpec);
+            byte[] encrypted = cipher.doFinal(toEncrypt.getBytes());
+
+            return DatatypeConverter.printBase64Binary(encrypted);
+
+        } catch (Exception ex) {
+            ex.printStackTrace();
```

Comment:
It is to be noted that though your printed a stacktrace, a user from outside the IDE will not be able to receive feedback on the error as it is not shown to the user (only on the console). Also, this means that your feature will "fail silently", which may not be the optimal design (to some extent, can be considered a bug during testing).